src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -1,43 +1,49 @@
 import functools
-import inspect
 import logging
 import time
-from typing import Any, Dict, List, Optional
 
-import slack_sdk
+from typing import Dict, List, Optional, Union
+
 from blockkit import Message, Section
-from slack_sdk.web.async_client import AsyncWebClient
+from slack_sdk.errors import SlackApiError
+from slack_sdk.web.client import WebClient
+from slack_sdk.web.slack_response import SlackResponse
+
 from tenacity import TryAgain, retry, retry_if_exception_type, stop_after_attempt
 
 from .config import SlackConversationConfiguration
 
-log = logging.getLogger(__name__)
+# List of Slack endpoints that require HTTP GET method
```

Comment:
It could be nice to make this into an `enum` since the endpoint strings are redefined in `make_call` functions.

Reply:
Ok, let me take a stab at it.

Reply:
Looks great